.ci/CI/src/Bootloader.jl

```diff
@@ -29,12 +29,21 @@ function parse_commandline()::Dict{String,Any}
         "--amdgpu"
         help = "Enable the generation of AMDGPU tests."
         action = :store_true
+        "--nointeg"
+        help = "Disable the generation of integration tests."
+        action = :store_true
         "--target-branch"
         help = "If target branch is set, does not read the target branch from a GitHub Pull Request which is set via environment variable `CI_COMMIT_REF_NAME`."
         arg_type = String
         "--project-path"
         help = "Set the path to the package folder of the package to be tested. Can also be set via the environment variable `CI_PROJECT_DIR`."
         arg_type = String
+        "--o-cpu-pipeline"
```

Comment:
Maybe these options deserve more suggestive names. I propose `--output-cpu` and `--output-gpu`. If I understand the functionality right, the job-scrips are the only output written to file, therefore, the word `pipeline` is not necessary here.

Reply:
Yes, we can do it. It makes sense.